src/actions/loginActions.js

```diff
@@ -10,6 +10,12 @@ export const loginUser = (response) => {
   };
 };
 
+export const cleanStore = () => {
```

Comment:
Entiendo lo que decis pero de la forma que lo hiciste es inmantenible, si a cada reducer hay q hacerle un clean.
Si realmente queres reiniciar la aplicacion podrias usar `window.location('/login')` y de esa forma se recarga la aplicacion. Sino queres recargar la aplicacion podes usar algo mas aproximado a esto http://stackoverflow.com/questions/35622588/how-to-reset-the-state-of-a-redux-store